draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1270,20 +1274,51 @@ The "rtp-mux-quic" string identifies RoQ:
   Specification:
   : This document
 
-## RoQ Error Codes
+## RoQ Error Codes Registry {#iana-error-codes}
+
+This document establishes a registry for RoQ error codes. The "RoQ Error Codes"
+registry manages a 62-bit space.
+
+The new error codes registry created in this document operates under the QUIC
+registration policy documented in {{Section 22.1 of !RFC9000}}. This registry
```

Comment:
I don't understand this part at all. There's a lot of cool stuff in 22.1, but is it all applicable to the RoQ error code registry?

Reply:
I tried to follow what [HTTP/3](https://datatracker.ietf.org/doc/html/rfc9114#name-error-codes) and [DNS over QUIC](https://datatracker.ietf.org/doc/html/rfc9250#name-dns-over-quic-error-codes-r) did. I'm not an expert on this and it may not be applicable, but to me, it seemed to make sense to follow what other application protocol mappings to QUIC do.